src/App.styles.tsx

```diff
@@ -4,9 +4,10 @@ import styled from 'styled-components';
 const SANDBOX_NAVBAR_HEIGHT = '3rem';
 const NAVBAR_HEIGHT = '4rem';
 
-export const MainContentContainer = styled.div<{ sandboxMode?: boolean }>`
+export const MainContentContainer = styled.div<{ sandboxMode?: boolean; fullHeight?: boolean }>`
     display: flex;
-    height: ${(props) => `calc(100vh - ${props.sandboxMode ? SANDBOX_NAVBAR_HEIGHT : NAVBAR_HEIGHT})`};
+    height: ${(props) =>
+        props.fullHeight ? `100vh` : `calc(100vh - ${props.sandboxMode ? SANDBOX_NAVBAR_HEIGHT : NAVBAR_HEIGHT})`};
```

Comment:
100%; not 100vh

Reply:
done 2e587316daeacd3e2dd7ecbb180b5a89bde74e7f